app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
```

Comment:
hmm https://github.com/Katello/katello/blob/master/app/models/katello/pool_with_quantities.rb indicates that we do have models here that are not rails related